x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/ServiceUtils.java

```diff
@@ -20,7 +21,7 @@
 
 import static org.elasticsearch.core.Strings.format;
 
-public class MapParsingUtils {
+public class ServiceUtils {
```

Comment:
Just figured this might be more appropriate as a catch all for util functions for the services.